src/examples/store.ts

```diff
@@ -5,6 +5,8 @@ import { routerReducer } from '../index';
 
 const combinedReducers = combineReducers({
     router: routerReducer({
+        activeTab: 1,
```

Comment:
we should start on the furthest left view, we may even want to make this configurable, 0

Reply:
Definitely need it configureable. Maybe pass down an initialTab prop?

Reply:
This is the pass in example. It currently works that way.